core/src/sql/order.rs

```diff
@@ -93,9 +93,8 @@ impl fmt::Display for Order {
 		if self.numeric {
 			write!(f, " NUMERIC")?;
 		}
-		match self.direction {
-			false => write!(f, " DESC")?,
-			true => (),
+		if !self.direction {
+			write!(f, " DESC")?
 		};
```

Comment:
I know this is fine as well, but have you considered putting this semi-colon after the `write!` macro? To me that would look a bit more natural.